controllers/parentRegistryController.js

```diff
@@ -23,6 +23,8 @@ module.exports = class ParentRegistry extends Abstract {
             if(typeof parent.type === "string") {
               parent.type = new Array(parent.type)
             }
+            parent.createdAt = new Date;
+            parent.updatedAt = new Date;
           })
 
           let addParentsQuery = await database.models["parent-registry"].insertMany(
```

Comment:
Using create instead of insertMany will do the job. Please check. Also we will need to write a script to update documents which don't have createdAt and updatedAt.